### ISSUE TITLE
In Samsung DEX, keybord uhid not work when main display off.

### ISSUE THREAD

**Issue by chieonol** [Genymobile/scrcpy#4747]
- [v] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [v] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows
 - scrcpy version: 2.4
 - installation method: Windows release...
 - device model: samsung galaxy s20 , s24+
 - Android version: 13,14

   **Describe the bug**
1. In dex, uhid doesn't take any key(including esc, space bar) when main display turned off.
2. If I turn on main display(by alt+p, alt+o dosen't work in dex), uhid works before screen off by timer. And if I turn off it with alt+p, uhid keyboard stop.
3. Maybe this problem only in samsung dex. (i tested lg desktop mode and no problem in display off)

**Comment by rom1v** [comment]
Does it also happen with a physical keyboard or mouse (connected physically or via bluetooth)?

**Comment by chieonol** [comment]
> Does it also happen with a physical keyboard or mouse (connected physically or via bluetooth)?

No. Only uhid happened. I used usb mouth and bluetooth keyboard.
